include/umf/memory_provider_ops.h

```diff
@@ -268,6 +268,20 @@ typedef struct umf_memory_provider_ops_t {
                             void *arg, size_t size,
                             umf_ctl_query_type_t queryType);
 
+    ///
+    /// @brief Retrieve properties of the memory allocation.
+    /// @param provider pointer to the memory provider
+    /// @param ptr pointer to the allocated memory
+    /// @param memory_property_id identifier of the memory property to retrieve
+    /// @param value [out] pointer to the preallocated variable where the value will be stored
+    /// @return UMF_RESULT_SUCCESS on success or appropriate error code on failure,
+    ///         UMF_RESULT_ERROR_INVALID_ARGUMENT if memory_property_id is invalid or value is NULL,
+    ///         UMF_RESULT_ERROR_NOT_SUPPORTED if the property is not supported by this provider.
```

Comment:
This is not true:
Please include tests to ensure that comment it true (as far i see we return EINVAL

Reply:
Especially please include also test with custom user provided provider, which support extra property which is user defined.